specification/sql/resource-manager/Microsoft.Sql/preview/2021-02-01-preview/Servers.json

```diff
@@ -523,7 +523,7 @@
           ],
           "type": "string",
           "x-ms-enum": {
-            "name": "ServerPublicNetworkAccess",
+            "name": "ServerNetworkAccessFlag",
```

Comment:
What is the reason to rename this enum name? as it create SDK breaking change

Reply:
The enum is being used in more places with this change, so we renamed it to make it more generic.